roles/cluster_infra/templates/outputs.tf.j2

```diff
@@ -3,6 +3,13 @@ output "cluster_gateway_ip" {
   value       =  openstack_compute_floatingip_associate_v2.login_floatingip_assoc.floating_ip
 }
 
+{% if cluster_ssh_private_key_file is not defined %}
+output "cluster_ssh_private_key" {
```

Comment:
Should this be marked as [sensitive](https://developer.hashicorp.com/terraform/language/values/outputs#sensitive-suppressing-values-in-cli-output)?

Reply:
Good idea, added